libs/dao/src/main/java/com/akto/dto/ApiCollection.java

```diff
@@ -25,6 +31,16 @@ public class ApiCollection {
     int urlsCount;
     public static final String VXLAN_ID = "vxlanId";
 
+    public enum Type {
+        API_GROUP
+    }
+
+    Type type;
+    public static final String _TYPE = "type";
+
+    List<CollectionCondition> conditions;
+    public static final String CONDITIONS_STRING = "conditions";
```

Comment:
What is `CollectionCondition` and why we adding more members to `ApiCollection`?

Reply:
collection condition is the filter which will be used to filter apis. We are adding more members to ApiCollection instead of creating another class, because, ApiCollections are used at some places